src/cljs/main/broadfcui/page/method_repo/method_config_importer.cljs

```diff
@@ -45,32 +48,43 @@
 
 (defn- create-import-form [state props this locals entity config? fields]
   (let [{:keys [workspace-id]} props]
-    [:div {}
+    [:div {:style {:display "flex"}}
      (when (:blocking-text @state)
        [comps/Blocker {:banner (:blocking-text @state)}])
-     [comps/EntityDetails {:entity entity}]
      (when (:allow-edit props)
-       [:div {:style {:margin "1em 0"}}
-        [:div {:style {:float "left" :width 290 :paddingRight "1em"}}
-         [comps/SidebarButton {:style :light :color :button-primary
-                               :text "Permissions..." :icon :settings
-                               :onClick #(modal/push-modal [mca/AgoraPermsEditor {:save-endpoint (endpoints/persist-agora-method-acl entity)
-                                                                                  :load-endpoint (let [[name nmsp sid] (map entity ["name" "namespace" "snapshotId"])]
-                                                                                                   (endpoints/get-agora-method-acl nmsp name sid config?))
-                                                                                  :entityType (entity "entityType") :entityName (mca/get-ordered-name entity)
-                                                                                  :title (str (entity "entityType") " " (mca/get-ordered-name entity))}])}]]
-        [:div {:style {:float "left" :width 290}}
-         [comps/SidebarButton {:style :light :color :exception-state
-                               :text "Redact" :icon :delete
-                               :onClick #(modal/push-modal [Redactor {:entity entity :config? config?
-                                                                      :on-delete (:on-delete props)}])}]]
-        (clear-both)])
-     [:div {:style {:border style/standard-line
-                    :backgroundColor (:background-light style/colors)
-                    :borderRadius 8 :padding "1em" :marginTop "1em"}}
-      [:div {:style {:fontSize "120%" :marginBottom "0.5em"}}
-       (if workspace-id "Import as:" "Export to Workspace as:")]
-      (map
+       [:div {:style {:flex "0 0 290px" :paddingRight "1rem"}}
+        (when-not config?
```

Comment:
right now, possible entities in the methods repo are "Workflow", "Task", and "Configuration". We're trying to do away with Tasks, but they still exist ... and this check allows duplication of both Tasks and Workflows. I think - and it's a PO call - we might only want to allow duplication on Workflows.